### ISSUE TITLE
Finalize Change Request Policy

### ISSUE THREAD

**Issue by acouch** [HHS/simpler-grants-gov#954]
### Summary

As part of the #465 , Simpler will adopt a change request policy that will inform HHS when changes are introduced. @jldroid19  and I propose a workflow that uses the existing release workflow and informs HHS of change details in a Jira ticket which links to public github release. This draft policy is located: https://app.gitbook.com/o/cFcvhi6d0nlLyH2VzVgn/s/v1V0jIH7mb7Yb3jlNrgk/engineering/security/draft-change-request-policy

This policy needs to be reviewed and accepted by HHS.

#### Sub-tasks

- [x] #955
- [ ] Confirm change request policy and workflow with HHS

### Acceptance criteria

- [ ] Change request policy documented and accepted

**Comment by sumiat** [comment]
currently blocked by HHS

**Comment by acouch** [comment]
This is blocked for this sprint and will be marked as blocked for the foresable future

**Comment by EOKENAVA** [comment]
@acouch says next step here is unknown/unclear. It should remain blocked but we need more conversations with HHS

**Comment by acouch** [comment]
Closing as this is no longer relevant as written.

**Comment by widal001** [comment]
Beep boop: Automatically setting the point and sprint values for this issue in project HHS/13 because they were unset when the issue was closed.
